test/testInlineScriptTags/myScript.js

```diff
@@ -1 +1 @@
-console.log('hi from myScript');
+console.log('hi from myScript </script>');
```

Comment:
Could you include a 2nd closing script tag to make sure all occurances are properly escaped?
```suggestion
console.log('hi from myScript </script></script>');
```

Reply:
Added